### PR TITLE
NEWS.md: update for flux-core v0.9.0

### PR DESCRIPTION
**Pull request by grondo**
I'm submitting the new NEWS entry for v0.9.0 as a PR so that other developers can push updates and edits directly here. Before merging we'll squash any edits.

I've already added entries for PRs #1513 and #1515

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,3 +1,95 @@
...
+    #1349, #1353, #1383, #1402, #1404, #1440, #1458, #1466, #1477)
+ * kvs: improved test coverage (#1291)
+ * Add const to message payload accessor functions (#1212)
+ * rename flux_mrpcf, flux_mrpc_getf to flux_mrpc_pack,unpack (#1338)
+ * cleanup bulk message hanglers in libflux (#1277)
+ * minor flux_msg_handler cleanup (#1171)
+ * broker: cleanup to prepare for dynamic service registration (#1189)
+ * broker: general cleanup (#1230, #1234, #1241)
+ * Change key lwj to jobid in all jsc/wreck messages (#1409)
+ * libjsc cleanup (#1374, 1395, #1509)
```
Comment (chu11) [inline NEWS.md]: missing ```#```

File: NEWS.md [hunk 2]
```diff
@@ -1,3 +1,95 @@
...
+ * testsuite updates (#1167, #1175, #1313, #1464, #1266)
+ * Internal libutil and libflux cleanup (#1319, #1283, #1229, #1231, #1166)
+ * build system cleanup (#1163, #1354, #1184, #1200, #1275, #1252)
+ * disable pylint by default (#1255, #1258)
+ * partial migration from json-c to jansson (#1501, 1508)
```
Comment (chu11) [inline NEWS.md]: missing ```#```

**Comment by grondo** [comment]
Thanks @chu11! Fixed.

I've rebased on current master and also tried to be more consistent throughout with backticks.

**Comment by garlick** [comment]
thanks guys! I'll go ahead and merge this.

**Comment by garlick** [comment]
> massive scalability improvement for libkz readers

_massive_ might be overstating this one a bit :-)

**Pull request merged**